### ISSUE TITLE
Missing figure on "The Figure-8 approach"

### ISSUE THREAD

**Issue by kswan** [backdrop-ops/docs.backdropcms.org#210]
There is a broken image after the first paragraph on https://docs.backdropcms.org/documentation/the-figure-8-approach

![image](https://user-images.githubusercontent.com/154391/193917986-46a9e7ff-8628-41e8-97ac-3b38eac2e684.png)

**Comment by bugfolder** [comment]
Here's how it looks on Safari on Mac (also Chrome and Firefox):

![figure](https://user-images.githubusercontent.com/29214084/193919808-ae9cfb2c-c281-4c6e-9c86-26d7108ff956.jpg)

What OS/browser are you using?

**Comment by kswan** [comment]
I tested on Edge, Chrome and Firefox and none show the image, but now I believe our corporate proxy is blocking the image. 

I noticed that the image source is https://i.postimg.cc/HWZtXXMT/config-deploy-strategy.png

Is it normal that d.b.o uses postimg.cc for image hosting?

Since this is clearly a problem on my end, I'll close the issue.

**Comment by kswan** [comment]
OK, thank you. I was digging through other pages trying to find whether images were hosted on-site or off-site. There aren't many images on d.b.o, but when I finally found a page with images (https://docs.backdropcms.org/documentation/user-guide) they were on-site.

**Comment by jenlampton** [comment]
I have added https://docs.backdropcms.org/file/11, and updated the src for the image to point to the local file. 

For history: this page was migrated from the b.org site, where there once was an image field on documentation nodes (back from before we had inline image support, and before the inclusion of file_entity). When we pulled across the content, we missed some of the images. 

Thank you for spotting the problem!
